VacaYAY/DataAccesLayer/Configuration/VacationRequestConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DataAccesLayer.Entities;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccesLayer.Configuration
+{
+    public class VacationRequestConfiguration : IEntityTypeConfiguration<VacationRequest>
+    {
+        public void Configure(EntityTypeBuilder<VacationRequest> builder)
+        {
+            builder.HasKey(v => v.Id);
+            builder.HasOne<Employee>(v => v.Employee).WithMany(e => e.VacationRequests).HasForeignKey(v => v.EmployeeId);
```

Comment:
This also. EF Core handles this automatically. Read more: https://learn.microsoft.com/en-us/ef/core/modeling/relationships